test/unit/services/service_listeners/attachment_link_header_updater_test.rb

```diff
@@ -22,7 +22,7 @@ class AttachmentLinkHeaderUpdaterTest < ActiveSupport::TestCase
     end
 
     context "when attachment doesn't belong to an edition" do
-      let(:attachment) { FileAttachment.new(attachable: PolicyGroup.new) }
+      let(:attachment) { FactoryBot.create(:file_attachment) }
```

Comment:
I'd prefer to explicitly set the `attachable` to something other than an Edition in this block. Not doing so makes it hard to see how this is working without digging into the `file_attachment` factory.